composer.scripts.json

```diff
@@ -0,0 +1,33 @@
+{
+  "scripts": {
+    "grumphp": [
+      "Composer\\Config::disableProcessTimeout",
+      "vendor/bin/grumphp run"
+    ],
+    "phpmetrics": [
+      "Composer\\Config::disableProcessTimeout",
+      "vendor/bin/phpmetrics --composer=false --report-html=/var/www/web/report/phpmetrics web/modules/custom"
+    ],
+    "phpstan": [
+      "Composer\\Config::disableProcessTimeout",
+      "vendor/bin/phpstan analyse"
+    ],
+    "phpcs": [
+      "Composer\\Config::disableProcessTimeout",
+      "vendor/bin/phpcs"
+    ],
+    "phpcbf": "vendor/bin/phpcbf",
```

Comment:
`"Composer\\Config::disableProcessTimeout"` is not included here for some reason

Reply:
Yes, because this script as usual works very fast